applications/luci-app-xfrpc/Makefile

```diff
@@ -0,0 +1,13 @@
+# This is free software, licensed under the Apache License, Version 2.0
+
+include $(TOPDIR)/rules.mk
+
+LUCI_TITLE:=LuCI Support for xfrpc -- c language frpc
```

Comment:
I think we do not need `-- c language frpc`. I think the line should look like:
`LUCI_TITLE:=LuCI Support for xfrpc`

Reply:
@feckert  already changed